gremlin-javascript/example/example.js

```diff
@@ -0,0 +1,143 @@
+  BOTH: direction.both,
+  IN: direction.in,
+  OUT: direction.out,
+  from_: direction.out,
+  to: direction.in,
+}
+const p = gremlin.process.P;
+const textp = gremlin.process.TextP;
+const pick = gremlin.process.pick;
+const pop = gremlin.process.pop;
+const order = gremlin.process.order;
+const scope = gremlin.process.scope;
+const t = gremlin.process.t;
+const cardinality = gremlin.process.cardinality;
+const CardinalityValue = gremlin.process.CardinalityValue;
+const serializer = gremlin.structure.io.graphserializer;
+const testing = Direction.to;
+
+async function main() {
+    await connectionExample();
+    await basicGremlinExample();
+    await modernTraversalExample();
+}
+
+
+
+async function connectionExample() {
+    // Connecting to the server
+    let dc = new DriverRemoteConnection('ws://localhost:8182/gremlin');
+    let g = traversal().withRemote(dc);
+
+    // Connecting and customizing configurations
+    g = traversal().withRemote(new DriverRemoteConnection('ws://localhost:8182/gremlin', {
+        mimeType: 'application/vnd.gremlin-v3.0+json',
+        reader: serializer,
+        writer: serializer,
+        rejectUnauthorized: false,
+        traversalSource: 'g',
+    }));
+    
+    // Cleanup
+    await dc.close();
+}
+
+async function basicGremlinExample() {
+    let dc = new DriverRemoteConnection('ws://localhost:8182/gremlin');
+    let g = traversal().withRemote(dc);
+
+    // Basic Gremlin: adding and retrieving data
+    const v1 = await g.addV('person').property('name','marko').next();
+    const v2 = await g.addV('person').property('name','stephen').next();
+    const v3 = await g.addV('person').property('name','vadas').next();
+
+
+    // Be sure to use a terminating step like next() or iterate() so that the traversal "executes"
+    // Iterate() does not return any data and is used to just generate side-effects (i.e. write data to the database)
+    await g.V(v1.value).addE('knows').to(v2.value).property('weight',0.75).iterate();
+    await g.V(v1.value).addE('knows').to(v3.value).property('weight',0.75).iterate();
+
+    // Retrieve the data from the "marko" vertex
+    const marko = await g.V().has('person','name','marko').values('name').toList();
+    console.log("name: " + marko[0]);
+
+    // Find the "marko" vertex and then traverse to the people he "knows" and return their data
+    const peopleMarkoKnows = await g.V().has('person','name','marko').out('knows').values('name').toList();
+    peopleMarkoKnows.forEach((person) => {
+        console.log("marko knows " + person);
+    });
+
+    await dc.close();
+}
+
+async function modernTraversalExample() {
+    /*
+    This example requires the Modern toy graph to be preloaded upon launching the Gremlin server.
+    For details, see https://tinkerpop.apache.org/docs/current/reference/#gremlin-server-docker-image and use
+    conf/gremlin-server-modern.yaml.
+    */
+
+    let dc = new DriverRemoteConnection('ws://localhost:8182/gremlin');
```

Comment:
this connection is not for modern graph

Reply:
Kept as is since current connection is able to load the Modern graph given that the user follows instructions in the ref link provided